addons/core/addon/components/loading-button/index.hbs

```diff
@@ -6,7 +6,7 @@
 <Hds::Button
   @text={{t 'actions.refresh'}}
   @color='secondary'
-  @icon={{if this.isLoading 'loading' 'reload'}}
+  @icon={{if this.toggleRefresh.isRunning 'loading' 'reload'}}
```

Comment:
where do we get this `isRunning` from?

Reply:
oh good question, that is [exposed by the task](https://ember-concurrency.com/docs/tutorial/refactor#:~:text=Version%202%3A%20Add%20a%20Loading%20Spinner%20(with%20Tasks))

Reply:
oh I see that we get this state from `ember-concurrency`

Reply:
Pretty neat!